test/comment-regex.js

```diff
@@ -5,15 +5,27 @@ var test = require('tap').test
   , generator = require('inline-source-map')
   , convert = require('..')
 
-function comment(prefix, suffix) {
-  var rx = convert.commentRegex;
-  return rx.test(prefix + 'sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiIiwic291cmNlcyI6WyJmdW5jdGlvbiBmb28oKSB7XG4gY29uc29sZS5sb2coXCJoZWxsbyBJIGFtIGZvb1wiKTtcbiBjb25zb2xlLmxvZyhcIndobyBhcmUgeW91XCIpO1xufVxuXG5mb28oKTtcbiJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFBQSJ9' + suffix)
+function comment(prefix, suffix, rx) {
+  return rx.exec(prefix + 'sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiIiwic291cmNlcyI6WyJmdW5jdGlvbiBmb28oKSB7XG4gY29uc29sZS5sb2coXCJoZWxsbyBJIGFtIGZvb1wiKTtcbiBjb25zb2xlLmxvZyhcIndobyBhcmUgeW91XCIpO1xufVxuXG5mb28oKTtcbiJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFBQSJ9' + suffix)
```

Comment:
I'm a bit worried about changing tests along with functionality since now we cannot be sure if the original behavior changed in an unintended way.
Is there a reason why this is needed?

Reply:
Left the original alone and added new ones for the newly supported formats (that all use `.test`)